invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/PatronCurrentLoans/__tests__/PatronCurrentLoans.js

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { generatePath } from 'react-router';
+import { Settings } from 'luxon';
+import { fromISO } from '../../../../../../common/api/date';
+import { BackOfficeURLS } from '../../../../../../common/urls';
+import PatronCurrentLoans from '../PatronCurrentLoans';
+
+Settings.defaultZoneName = 'utc';
+const d = fromISO('2018-01-01T11:05:00+01:00');
```

Comment:
I wouldn't use single letter variables, not even for tests.

Reply:
copy-paste results :)